### PR TITLE
adding wokwi design submission guide

### PR DESCRIPTION
**Pull request by kdotom**
adding submission guide for wokwi designs

### CODE REVIEW

File: content/guides/wokwi-submit-design/_index.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+Once the shuttle run has ended, to update your documentation:
+
+* Follow this guide: [Updating Docs Post-Shuttle](#).
+
+## Get Inspired by Other Designs
+
+Looking for ideas or want to see what others have created?
+
+* Browse past runs: [Tiny Tapeout Runs](#).
+* Check out a sample datasheet: [Sample Datasheet PDF (TT07)](#).
```
[the remark this answers is not in the document]
Comment (kdotom) [reply content/guides/wokwi-submit-design/_index.md]: updated

File: content/guides/wokwi-submit-design/_index.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+## About Efabless
+
+Efabless is our manufacturing partner, helping to make chip fabrication accessible. [Learn more about them here](#).
+
+## Updating Documentation After the Shuttle Ends
+
+Once the shuttle run has ended, to update your documentation:
+
+* Follow this guide: [Updating Docs Post-Shuttle](#).
```
[the remark this answers is not in the document]
Comment (kdotom) [reply content/guides/wokwi-submit-design/_index.md]: updated

File: content/guides/wokwi-submit-design/_index.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+### Tips
+
+* Double-check your design before submitting to avoid the need for updates.
+* Stay aware of submission deadlines to ensure your design is included.
+
+## Purchasing a PCB
+
+If you'd like to receive a physical PCB with your chip:
+
+* Pre-purchase your PCB here: [Tiny Tapeout Prepurchase](#).
```
[the remark this answers is not in the document]
Comment (kdotom) [reply content/guides/wokwi-submit-design/_index.md]: updated

File: content/guides/wokwi-submit-design/_index.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+  
+> **Note:** Only the first 100 boards are subsidized by Efabless, so act fast!
+
+## About Efabless
+
+Efabless is our manufacturing partner, helping to make chip fabrication accessible. [Learn more about them here](#).
```
[the remark this answers is not in the document]
Comment (kdotom) [reply content/guides/wokwi-submit-design/_index.md]: updated

File: content/guides/wokwi-submit-design/_index.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+## Updating Documentation After the Shuttle Ends
```
[the remark this answers is not in the document]
Comment (kdotom) [reply content/guides/wokwi-submit-design/_index.md:107]: updated

File: content/guides/wokwi-submit-design/_index.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+{{< figure src="images/submit_design_2.png" >}}
+{{< figure src="images/submit_design_3.png" >}}
+
+* Paste it into the field and click **"Submit"**.
+
+### Complete the Submission Process
+
+* You have now submitted your design, but it’s not yet part of the tapeout. 
+
+{{< figure src="images/submit_design_4.png" >}}
```
[the remark this answers is not in the document]
Comment (kdotom) [reply content/guides/wokwi-submit-design/_index.md:62]: updated: file replaced.

File: content/guides/wokwi-submit-design/_index.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+* Click **"Submit a New Revision"** to add your design to the shuttle.
+* Your design will then be added to the Tapeout.
+  
+> **Note:** You'll need to submit any revisions if changes are made before final submission.
+
+### Apply Your Free Tile Coupon
```
[the remark this answers is not in the document]
Comment (kdotom) [reply content/guides/wokwi-submit-design/_index.md]: updated

File: content/guides/wokwi-submit-design/_index.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+
+* During the submission, you'll have the option to apply a coupon for your free tile. (Feel free to ask for a coupon code.)
```
[the remark this answers is not in the document]
Comment (kdotom) [reply content/guides/wokwi-submit-design/_index.md]: updated.

**Comment by mattvenn** [comment]
Thanks!

**Pull request merged**